### DISCUSSION TITLE
DIPY open lab meetings -- fall 2019

### DISCUSSION THREAD

**Discussion by arokem** [dipy/dipy#2357] [2. Announcements]
The schedule for upcoming lab meetings is:

September 10th, 1600 UTC: new website (@Garyfallidis)
September 24th, 1600 UTC: 1.1 Release (@skoudoro)
October 8th, 1600 UTC: New encoding schemes (@RafaelNH leads the discussion)
October 22nd, 1600 UTC: Workflows (@skoudoro)
November 5th, 1700 UTC:  FWDTI (@RafaelNH)
November 19th, 1700 UTC: Registration (@skoudoro)
December 3rd, 1700 UTC: Horizon (@Garyfallidis)

All meetings will be held through Zoom at the following link: https://washington.zoom.us/j/812354748.

Meetings will be 60 minutes at the most.

**Comment by arokem** [comment]
As you can see, there are plenty of open slots. If there is something in particular you would like to discuss, please ping us here and we can edit the schedule to add it.

**Comment by skoudoro** [comment]
I would like to discuss:

- Workflow framework
- Registration framework
- Release 1.1 in October

But I do not know when yet 😄. Need to think about it

**Comment by Garyfallidis** [comment]
Need to setup a plan for the new encoding schemes and update our gradient table and some of the API too.

**Comment by arokem** [comment]
@skoudoro : presumably one of the early slots for the discussion of 1.1? How about 9/24 for that? 

@Garyfallidis : would be great to devote a meeting to new encoding schemes. I know that @RafaelNH is also interested in that! Would 10/8 work for that discussion?

**Comment by skoudoro** [comment]
> presumably one of the early slots for the discussion of 1.1? How about 9/24 for that?

:+1: Let's do that :+1:

**Comment by RafaelNH** [comment]
> @skoudoro : presumably one of the early slots for the discussion of 1.1? How about 9/24 for that?
> 
> @Garyfallidis : would be great to devote a meeting to new encoding schemes. I know that @RafaelNH is also interested in that! Would 10/8 work for that discussion?

Works for me!

**Comment by arokem** [comment]
Reminder: we will have a lab meeting tomorrow at 9 AM PST, noon EST, 16 UTC. 

@Garyfallidis will give us a run-down of the new website and its capabilities. 

We can also spend a bit of time planning our meetings for the rest of the Fall. 

The call link is: https://washington.zoom.us/j/812354748

**Comment by arokem** [comment]
@RafaelNH : I've assigned you to lead the discussion on encoding schemes on 10/8! Let me know if that doesn't work for you.

@ecaruyer : We are thinking about new encoding schemes for DIPY and we'd like to have a discussion about this at the DIPY lab meeting on 10/8 (1600 UTC). Would be great if you could join us for this meeting. We know you have thought about this a lot, so your feedback would be valuable.

**Comment by arokem** [comment]
Schedule update through the end of 2019. Take a look!

**Comment by ecaruyer** [comment]
> We are thinking about new encoding schemes for DIPY and we'd like to have a discussion about this at the DIPY lab meeting on 10/8 (1600 UTC). Would be great if you could join us for this meeting. We know you have thought about this a lot, so your feedback would be valuable.

Sounds great! Oct 8th 1600 UTC works for me, I put it on my calendar.

**Comment by skoudoro** [comment]
Reminder: we will have a lab meeting tomorrow at 9 AM PST, noon EST, 16 UTC.

The topic is DIPY 1.1 Release Plan. We will go through some PR's and establish a priority list for the future release in October. If you have any request, be present tomorrow!

The call link is: https://washington.zoom.us/j/812354748

**Comment by RafaelNH** [comment]
Reminder: Today, October 8th, we have a lab meeting at 1600 UTC.

I will be leading a discussion about new diffusion encoding schemes.

**Comment by arokem** [comment]
Reminder: tomorrow’s meeting a at 9 am PT/ noon ET/ 1600 CET. @skoudoro will be leading a discussion on workflows.

**Comment by RafaelNH** [comment]
Reminder: Tomorrow, November 4th, we have a lab meeting at 1600 UTC.

@mvgolub and I will give an update of the work done in free water DTI algorithms for single-shell algorithms.

**Comment by arokem** [comment]
This should be 1700 UTC moving forward. I also just updated the comment at the top of this issue.

**Comment by skoudoro** [comment]
Reminder: Tomorrow, November 19th, we have the online DIPY lab meeting at 1700 UTC.

The topic: Quick overview and discussion about DIPY registration framework. You might be interested @Borda , @grlee77, @omarocegueda, @d-v-b.

The call link is: https://washington.zoom.us/j/812354748

**Comment by skoudoro** [comment]
Reminder: Today, December 3rd, we have the online DIPY lab meeting at 1700 UTC.

The topic: Quick overview about DIPY Horizon by @Garyfallidis . You might be interested @giulia-berto.

The call link is: https://washington.zoom.us/j/812354748

**Comment by arokem** [comment]
Closed through #2024
